### ISSUE TITLE
pom.xml file will be configured when another pom file specified to mvn command line.

### ISSUE THREAD

**Issue by m-moris** [microsoft/azure-maven-plugins#757]
### Plugin name and version

```
        <groupId>com.microsoft.azure</groupId>
        <artifactId>azure-webapp-maven-plugin</artifactId>
        <version>1.7.0</version>
```
### Expected behavior

I have two pom files in same directory. This one is `pom.xml` for Windows configuration and another one is `pom.linux.xml` for Linux configuration.

I executed following command, The `pom.xml` file will be configured. 

```
mvn -f pom.linux.xml azure-webapp:config
```

### Actual behavior

The `pom.linux.xml` file must be configured.

### Steps to reproduce the problem

1.  Preparing some pom.xml
2.  `mvn -f pom.xml azure-webapp:config`
3. copy `pom.xml` to `pom.linux.xml`
4. `mvn -f pom.linux.xml azure-webapp:config`

Thanks.

**Comment by Flanker32** [comment]
@m-moris Thanks for your report, for now webapp plugin will only config `pom.xml` in current folder, we will fix this bug in next release, sorry for the incovenice.

**Comment by Flanker32** [comment]
Fixed and released in [webapp maven plugin 1.8.0](https://search.maven.org/artifact/com.microsoft.azure/azure-webapp-maven-plugin/1.8.0/maven-plugin)
